internal/handlers/pg/pg.go

```diff
@@ -16,22 +16,29 @@
 package pg
 
 import (
+	"context"
+	"sync"
+
 	"go.uber.org/zap"
 
 	"github.com/FerretDB/FerretDB/internal/clientconn/connmetrics"
 	"github.com/FerretDB/FerretDB/internal/handlers"
 	"github.com/FerretDB/FerretDB/internal/handlers/pg/pgdb"
+	"github.com/FerretDB/FerretDB/internal/util/lazyerrors"
 	"github.com/FerretDB/FerretDB/internal/util/state"
 )
 
 // Handler implements handlers.Interface on top of PostgreSQL.
 type Handler struct {
 	*NewOpts
+
+	rw    sync.RWMutex
+	pools map[string]*pgdb.Pool
```

Comment:
The idea is that none of `Msg` methods should request this map directly, they should always call `h.DBPool(ctx)` instead. However, in principle such call is possible. Should we leave a comment that it is expected to use `h.DBPool`, but not this map + mutex directly? Or is it obvious?

Reply:
I will add a comment in #1725, but someone should _really_ try to use that map…